berkeley-function-call-leaderboard/eval_checker/eval_runner_helper.py

```diff
@@ -697,6 +715,22 @@ def update_leaderboard_table_with_score_file(leaderboard_table, score_path):
                 }
 
 
+def oss_file_formatter(input_file_path, output_dir):
```

Comment:
Can we do a simple assert on the # of rows in the file? If there is a mismatch, we should flag it.

Reply:
yea sounds good. Let me add that